src/logion/model/supportedaccountid.model.ts

```diff
@@ -1,52 +1,82 @@
 import { components } from "../controllers/components.js";
 import { Column } from "typeorm";
+import { ValidAccountId, AnyAccountId, AccountType, AccountId } from "@logion/node-api";
 
 export type SupportedAccountId = components["schemas"]["SupportedAccountId"];
```

Comment:
We should be able to remove this export.

Reply:
Indeed.